README.md

```diff
@@ -1,6 +1,15 @@
-# rekor-monitor
+# Rekor Log Monitor
+
+Rekor Log Monitor provides an easy-to-use monitor to verify log consistency,
+that the log is immutability and append-only. Monitoring is critical to
+the transparency log ecosystem, as logs are tamper-evident but not tamper-proof.
+
+To run, simply fork this repository, and you'll begin to monitor Rekor. The monitor
+runs on GitHub Actions.
```

Comment:
forks need a user to enable actions in the actions tab I believe?

Reply:
Yes, correct, although i think the default is actions enabled?

Reply:
this is what I see
![image](https://user-images.githubusercontent.com/1304826/199579973-ae1c9b50-eff7-4e4b-8b67-11b14592e6c6.png)

Reply:
this is what I see
![image](https://user-images.githubusercontent.com/1304826/199579973-ae1c9b50-eff7-4e4b-8b67-11b14592e6c6.png)

Reply:
Ah thanks, maybe I have some other setting or something's changed? I'll add a note in the readme.

Reply:
I would maybe add in something like 
```
after you fork, ensure actions are enabled to allow the monitor to run
``